src/studio/ui/StudioPanel.java

```diff
@@ -2182,14 +2242,19 @@ private void processK4Results(K.KBase r) throws c.K4Exception {
         else {
             // Log that execute was successful
         }
+        tabbedPane.setSelectedIndex(tabbedPane.getTabCount()-1);
```

Comment:
For table, list and dictionary results, we create 2 tabs, one with a table view and one with a text view. This will focus on the tab with the text view, which is often not what the user wants to see but it was added to allow the user to copy the content as text (e.g. to copy the text representation of a list/dict into another query).

Reply:
ok - will make do "-2"

Reply:
Actually it's not always 2 either. It depends on the type of the result of the query.

Reply:
Good point - let me update